docs/register-create-stake-phase-2.md

```diff
@@ -0,0 +1,227 @@
+ownership of the associated Bitcoin address. This will be generated during the 
+phase-1 creation of stake process. To see the full message, see the 
+[`ProofOfPossessionBTC` message](./proto/babylon/btcstaking/v1/pop.proto).
+
+`inclusion_proof` (Optional): 
+A Merkle proof showing that the staking transaction is included in the 
+Bitcoin chain. This field is optional but recommended. To see the full message, 
+see the [`InclusionProof` message](./proto/babylon/btcstaking/v1/btcstaking.proto).
+
+If not provided during submission, it must be submitted later using the 
+`MsgAddBTCDelegationInclusionProof` message. To see the full message, see the 
+[`MsgAddBTCDelegationInclusionProof` message](./proto/babylon/btcstaking/v1/tx.proto).
+
+`Other General Fields`:
+- `staking_value`: The amount of BTC locked for delegation (in satoshis).
+- `staking_time`: The timelock period for the staking transaction (in blocks).
+- `unbonding_time`: The timelock period for unbonding (in blocks).
+
+## 3. Stake creation and registration
+### 3.1. Registering a phase-1 stake
+
+To register a Bitcoin stake that is already on Bitcoin to 
+the Babylon chain (e.g., a phase-1 stake), you need to submit a 
+`MsgCreateBTCDelegation` with a valid inclusion proof. This requires your 
+staking transaction to already be confirmed on the Bitcoin network. You can do 
+this either manually through the Bitcoin network or using the staker library.
+
+1. Create `MsgCreateBTCDelegation` with the inclusion proof filled as defined 
+above and submit `MsgCreateBTCDelegation` to the Babylon network's btcstaking 
+module. You will then wait for the covenant committee to submit their signatures.
+
+
+Alternatively, you can use the CLI to register phase-1 stake:
+
+```bash
+babylond tx btcstaking create-btc-delegation [btc_pk] [pop_hex] [staking_tx] [fp_pk] [staking_time] [staking_value] [slashing_tx] [delegator_slashing_sig] [unbonding_tx] [unbonding_slashing_tx] [unbonding_time] [unbonding_value] [delegator_unbonding_slashing_sig]
+```
+
+2. You can check the status of your delegation by querying the babylon node 
+like this.
+
+```bash
+babylond query btcstaking btc-delegations <staking-tx-hash>
+```
+
+The above command will return the status of your delegation (pending, active, 
+unbonding, unbonded, any).
+
+### 3.2. Creating new stakes
+
+To create a new stake, you need to submit a `MsgCreateBTCDelegation` with an 
+empty inclusion proof to the Babylon network. The message should include:
+
+- Your unsigned staking transaction
+- Pre-signed slashing transaction
+- Unsigned unbonding transaction
+- Pre-signed unbonding slashing transaction
+- Proof of Possession (PoP)
+- Other required fields (staking amount, timelock periods, etc.)
+
+Please note that this flow can also be referred to as the Expression of Interest 
+(EOI) workflow. More information can be found [here](./staking-script.md).
+
+
+If you would like to create a new stake using the CLI, you can use the 
```

Comment:
No need for CLI instructions in this spot. We can note CLI details as an option to transmit `MsgCreateBTCDelegation` in Section 2